app/models/spree/user.rb

```diff
@@ -142,6 +142,11 @@ def clear_spree_api_key!
       save!
     end
 
+    def toggle_api_key_view!
+      self.show_api_key_view = !show_api_key_view
+      save!
+    end
+
```

Comment:
If we use `update` then we don't need this.

Reply:
All right. :) I thought it should be similar to the other api actions.